doc/manual/source/physics/active_particles.rst

```diff
@@ -0,0 +1,146 @@
+.. _active_particles:
+
+
+Active Particles
+============================================
+
+Active Particles (APs) were introduced into Enzo in version 2.6. They were originally part
+of the Enzo-3.0 effort and were developed by a number of authors including Nathan Goldbaum,
+Matt Turk, John Wise, John Regan, Oliver Hahn, Greg Meece, Brian Crosby and others. Their design philosophy was that the
+APs would be highly extensible, have robust feedback capabilities and be very much object orientated.
+They were designed to replace the star object class.
+
+Compared to the star objects the APs can more easily connect with the radiative trasfer solver this making the
+framework more extensible in that context. Other feedback mechanisms (e.g. mechanical and thermal) are
+also easily implemented. 
+
+Using Active Particles
+______________________
+
+To implement the active particles framework in any Enzo run is very simple. Add the following line to the
+parameter file that you are using to (re)start Enzo.
+``AppendActiveParticleType = <YourActiveParticleType>``
+where ``<YourActiveParticleType>`` is one of the currently available AP types. 
+
+Types of Active Particles
+_________________________
+
+At the moment there are a number of AP type implementations. However, with the exception of the "SmartStar"
+AP type none of the implementations have been (robustly) tested.
+
+* SmartStar
+* Accreting Particle
```

Comment:
This is a comment related to the Enzo code file not the documentation. In the file 'ActiveParticle_AccretingParticle.h' and 'ActiveParticle_SmartStar.h' there is a class  'DepositMass' defined. One of the argument for this class is 'GalaxyParticleID' in both which I guess is an error getting carried over due to using the modified  'ActiveParticle_GalaxyParticle.h' for these two cases.

Reply:
OK. That should be a separate PR though. Can you raise an issue on it or open a PR?